pycozmo/client.py

```diff
@@ -440,6 +440,6 @@ def get_anim_names(self) -> set:
     def anim_names(self) -> set:
         return self.get_anim_names()
 
-    def play_audio(self, audio):
-        self.audio.play_file(audio)
+    def play_audio(self, fspec: str) -> audio.AudioManager:
+        self.audio.play_file(fspec)
```

Comment:
`audio` is the name of a module in the global scope - renamed to avoid shadowing it.